util/perl/TLSProxy/Proxy.pm

```diff
@@ -321,8 +346,6 @@ sub clientstart
 
     die "No progress made" if $ctr >= 10;
 
-    END:
-    print "Connection closed\n";
```

Comment:
I wonder why the "Connection closed" is no longer in the V=1 trace.

Reply:
No "Server connection closed" or "Client connection closed" either?

Reply:
so, why is the new print to STDERR, and guarded by self->debug ?

Reply:
You know, good point.  I'll add that back...

Reply:
Yes, not a single "Server|Client connection closed",
only one "Error on client end: Connection reset by peer"